.github/workflows/ci.yml

```diff
@@ -26,13 +26,19 @@ jobs:
           # packages in it.
           - sombrero@2021-08-16
           - example@2021-08-16
-        reframe_version:
-          # In README.md we claim compatibility with v3.11.0
-          - v3.11.0
-          # Our ReFrame config files are currently not compatible with `master`
-          # - master
+        # Compatible ReFrame versions are set in pyproject.toml
     steps:
       - uses: actions/checkout@v3
+      - uses: actions/setup-python@v4
+        with:
+          python-version: '3.10.6'
```

Comment:
python 3.6 is [not available](https://github.com/actions/setup-python/issues/561) for Ubutu 22 on GHA. I think the earliest I could go was 3.8. I probably left it at `3.10.6` because that was what I had installed and I wanted to make sure I'm not getting issues from different python versions. We can run CI on Ubuntu 20 if you want to test on 3.6?

Reply:
Actually, no, 3.7.12 is the oldest they have for ubuntu 22.04 https://raw.githubusercontent.com/actions/python-versions/main/versions-manifest.json

Reply:
Leaving it as 3.10 for now. Pip in python 3.6 is failing to install reframe